### ISSUE TITLE
can't use if statement in the panels component

### ISSUE THREAD

**Issue by fithrahs** [naver/egjs-flicking#650]
## Description

hi, i've tried use v-if in the card panel

```
<Flicking :options="{ horizontal: true, align: 'prev', circular: true }" :plugins="plugins" class="mb-1">
  <div class="card-panel" v-if="false">
  </div>
  <div class="card-panel">
  </div>
  <div class="card-panel">
  </div>
</Flicking>
```

but i got this error
![image](https://user-images.githubusercontent.com/85327078/162418444-01cd694d-4e9f-4451-93ee-e66f8c614d9e.png)

and  i can't slide/scroll the flicking component, if i try to slide i got this error
![image](https://user-images.githubusercontent.com/85327078/162418860-7af5f9a9-3ce9-4c7b-98bd-95ad6283f440.png)

is possible to use if statement in the panel component?

**Comment by WoodNeck** [comment]
Hello @fithrahs! I could reproduce this issue only in vue3-flicking, so I guess you're using Vue@3.
I've created a PR(#651) that fixes this issue, I'll leave a comment after it's released :)

**Comment by WoodNeck** [comment]
I've just released a new version(`v4.6.0`) that fixes this issue, please try that :)
